lazer/sdk/js/src/socket/websocket-pool.ts

```diff
@@ -104,7 +121,9 @@ export class WebSocketPool {
         rws.onError = config.onError;
       }
       // Handle all client messages ourselves. Dedupe before sending to registered message handlers.
-      rws.onMessage = pool.dedupeHandler;
+      rws.onMessage = (data) => {
+        void pool.dedupeHandler(data);
```

Comment:
Personally I don't like using `void` to discard errors because it's not clear that discarding errors is the intention, when I choose to discard errors I prefer to write it as:

```ts
pool.dedupeHandler(data).catch(() => { /* no-op on error */ });
```

That way it's very obvious to future readers that the intention was to discard errors, and it wasn't an accident.

Also I'd probably just at least log out the error message rather than swallowing it silently; unless you expect for errors to happen and if they do it's not important, you probably at least want a message in the event that something breaks unexpectedly.